Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8/ruby-26
+FROM ruby:alpine as builder
 ENV HEIMDALLTOOLS_VERSION 1.3.6
```

Comment:
This shouldn't be hardcoded to a version, I'd suggest we just drop this ENV completely.